code/modules/fissionreactor/fuelmaker.dm

```diff
@@ -0,0 +1,115 @@
+/*
+in this file:
+the machine which makes fuel rods have things in them.
+*/
+
+//because radon is a gas, we need to interface with gasses. yeah, this kind of sucks, but what are you gonna do? (inb4 make better code lol)
+/obj/machinery/atmospherics/unary/fissionfuelmaker
+	name="isotopic separational combiner." //just about the most technobable you could get.
+	var/datum/reagents/held_elements=new /datum/reagents
+	use_power = MACHINE_POWER_USE_IDLE
+	idle_power_usage = 200
```

Comment:
You intend to have this use a significant amount of power while idle but hardly any while active?

Reply:
i don't really know what any of it means